src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -641,6 +687,47 @@ internal static string NormalizeFilePath(string path)
             return Path.GetFullPath(path);
         }
 
+        /// <summary>
+        /// Determine the execution policy based on the supplied string.
+        /// If the string doesn't match to any known execution policy, set it to incorrect.
+        /// </summary>
+        /// <param name="_executionPolicy">The value provided on the command line.</param>
+        /// <returns>The execution policy.</returns>
+        private static ParameterBitmap GetExecutionPolicy(string? _executionPolicy)
+        {
+            if (_executionPolicy is null)
+            {
+                return ParameterBitmap.EPUndefined;
+            }
+
+            if (MatchSwitch(_executionPolicy, "remotesigned", "remotesigned"))
```

Comment:
Would this be better as a Dictionary that maps the string to the bitmap value?

Reply:
This allowed me to reuse code that we use already for parameter recognition.

Reply:
Given that you don't need to check on a substring match, maybe just use `_executionPolicy.Equals("remotesigned", StringComparison.OrdinalIgnoreCase)`?